### ISSUE TITLE
Flight log seems to wipe unexpectedly and frequently

### ISSUE THREAD

**Issue by HelloMiko** [pioneerspacesim/pioneer#5158]
### Observed behaviour
After playing for half hour, recording multiple logs (attempting to begin a story), I noticed the 'edit' button of the Flight log page began to move off the screen so I couldn't click it anymore. I tried to fix by changing the resolution, saving the game, and performing a restart, which worked, although all Flight logs were wiped upon restart.

### Expected behaviour
Flight logs remain forever. Flight log UI remains stationary.

### Steps to reproduce
Potentially change resolution and perform restart but your mileage may vary.

__My pioneer version (and OS):__ 20210327, Win 10 64 bit

**Comment by impaktor** [comment]
Ouch. Sounds like two bugs here. One UI (not so serious), and the wiping of the log, which someone else mentioned in IRC some time ago (unless it was you). 

The latter bug does worry me, and I don't know how to debug it.

Also, cudos on actually "role playing"/writing story into the log, it's exactly how we hoped it would be used :)

**Comment by HelloMiko** [comment]
Yes I thought a record of exploration and isolation would be quite nice to read through after a couple of years in space. It's a very cool feature :)

Anyway, I was playing on 1920x1080 originally then changed it to one of the 4:3 resolutions... cant remember which exactly.

**Comment by jimishol** [comment]
Conifrmed. Edit and Delete buttons does not exist anymore. So, one can only insert new records in Custom Logs but not delete or edit them.
